sqldelight-gradle-plugin/build.gradle

```diff
@@ -142,7 +178,6 @@ tasks.named('check') {
 }
```

Comment:
I didn't add instrumentationTest here to keep existing behavior which also tests on-device only on CI.